proposals/61:lo-bundle-cost/Makefile

```diff
@@ -0,0 +1,13 @@
+
+lower-bundle-cost.json:
+	jq '.tx.body.messages[0].content | .changes[0].value |= fromjson' tx-61.json >$@
+
+hash=6EA7B46DFDD0973338BB50781B33D651FE0E8EE8ABF4D7BCB1BF8235334AAFE0
```

Comment:
very cool to pull from an archive node.

can we make this easier to work with? e.g. a default ARCHIVE_NODE value?

how does determine the correct hash value?

Reply:
I added the relevant query commands to go from the proposal number 61 to the tx hash. Overkill?